CMSgov/mint-app
# [EASI-4353] Discussion Reply Translated Audit Meta Data

# EASI-4353


## Changes and Description

- Discussion Reply has it's own audit meta data
   - discussion content
   - discussion id
   - number of replies for that discussion at that moment in tiem
 


## How to test this change
1. Make a discussion
2. Make multiple replies
3. Translate the audits
4. Ensure the new meta data is present on all discussion replies
5. Ensure the new meta data shows the correct number of replies regardless of the time it was translated (eg the first one says 1, the second says 2 etc) 



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
